contracts/src/blobstream/blobstream_contract.ts

```diff
@@ -47,18 +47,16 @@ export class BlobstreamProcessor extends SmartContract {
 
     @method async update(admin: PrivateKey, blobstreamProof: BlobstreamProofType, path: BlobstreamMerkleWitness) {
         blobstreamProof.verify()
-        let leafIndex = this.currentLeafIndex.get();
-        this.currentLeafIndex.requireEquals(leafIndex);
+        let leafIndex = this.currentLeafIndex.getAndRequireEquals();
 
-        let commitmentsRoot = this.commitmentsRoot.get();
+        let commitmentsRoot = this.commitmentsRoot.getAndRequireEquals();
         this.commitmentsRoot.requireEquals(commitmentsRoot);
```

Comment:
@45930 do we still need this line after you change?

Reply:
Ah, no.  `get()` and `requireEquals()`  can be replaced with `getAndRequireEquals()`

Reply:
does it look correct now?

Reply:
Yeah, thanks!